src/resources/destinations.tsx

```diff
@@ -27,14 +29,26 @@ import {
   useNotify,
   useRefresh,
   useTranslate,
+  useTheme,
+  DateFieldProps,
 } from "react-admin";
 
 import { DATE_FORMAT } from "../components/date";
 
 const DestinationPagination = () => <Pagination rowsPerPageOptions={[10, 25, 50, 100, 500, 1000]} />;
 
-const destinationRowSx = (record: RaRecord) => ({
-  backgroundColor: record.retry_last_ts > 0 ? "#ffcccc" : "white",
+const destinationRowSxLight = (record: RaRecord) => ({
+  backgroundColor: record.retry_last_ts > 0 ? "#ffcccc" : undefined,
```

Comment:
```suggestion
  backgroundColor: record.retry_last_ts > 0 ? "error.light" : undefined,
```

I would recomend use named color. This eliminates the need to query the theme and is compatible with all themes.

The list of color names are there: https://mui.com/material-ui/customization/default-theme/